server/mediaserver.go

```diff
@@ -489,12 +310,84 @@ func endRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 		if monitor.Enabled {
 			monitor.LogStreamEndedEvent(cxn.nonce)
 		}
+		cxn.eof <- struct{}{}
 		delete(s.rtmpConnections, mid)
 
 		return nil
 	}
 }
 
+func (s *LivepeerServer) startSession(cxn *rtmpConnection) *BroadcastSession {
+
+	mid := rtmpManifestID(cxn.stream)
+	cpl := cxn.pl
+	var sess *BroadcastSession
+
+	// Connect to the orchestrator. If it fails, retry for as long
+	// as the RTMP stream is alive; maybe the orchestrator hasn't
+	// received the block containing the job yet
```

Comment:
I think we can update the last part of this comment since orchestrators no longer wait for blocks containing jobs

Reply:
Fixed